lib/SIL/IR/Bridging.cpp

```diff
@@ -187,6 +188,7 @@ Type TypeConverter::getLoweredCBridgedType(AbstractionPattern pattern,
     case SILFunctionType::Representation::Thin:
     case SILFunctionType::Representation::Method:
     case SILFunctionType::Representation::ObjCMethod:
+    case SILFunctionTypeRepresentation::CXXMethod:
```

Comment:
Not part of this PR, but the comment above looks a tad outdated.

```
// Functions that are already represented as blocks or C function pointers
// don't need bridging.
```

Potentially want to update that (or remove it) since you are updating nearby stuff.

Reply:
Hmm. Is this comment saying "don't need to be bridged here" or "don't need bridging in general"? The latter is outdated but the former seems OK. Also happy to remove it.